wordpress-mobile/WordPress-Android
# Jetpack focus jetpack powered bottom sheet

This PR introduces **Jetpack powered** bottom sheet to be invoked from Jetpack badges, banners and tabs.  

Fixes: #16974 & #16978

**NOTE**:  Consider this as part 1 to keep the PR manageable.  It is only hooked to Jetpack badge on home tab for testing.  Although a full screen version is in implementation, it is not complete and not hooked to Reader, Stats and Notifications tabs yet.  It'll be updated in the next iteration to proper layout and styles subject to final discussions.

https://user-images.githubusercontent.com/990349/179146377-61ffb43d-d252-4754-99ed-172a593f6fc2.mp4


To test:

Setup:

- Launch WordPress app
- Go to App Settings (Tap on Avatar at the top right hand corner on My Site -> Find App Settings)
- Select Debug Settings
- Find JetpackPoweredFeatureConfig under Features in development as shown on this PR #16908
- Re-launch app (scroll down if necessary)

Test 1:

- Make sure the Jetpack powered badge appears on the home tab as shown on this PR #16901 
- Tap on the Jetpack powered badge, Ensure a bottom sheet appears as shown in video above
- Observe Logo animation 

Test 2:

- Tap on Get the Jetpack app button, it should open Play Store for Jetpack app

Test 3:

- Return to home, tap on badge again.  When bottom sheet opens 
- Try dragging down and ensure it closes.
- Return to home, tap on badge again.  When bottom sheet opens, tap anywhere outside, ensure bottom sheet closes.


## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
Existing unit tests

PR submission checklist:
